README.md

```diff
@@ -69,6 +69,17 @@ const q = url.searchParams.get("q");
 console.log(q); // query
 ```
 
+## Get multiple parameters as array from URL
+
+Use [`URL`](https://developer.mozilla.org/docs/Web/API/URL/URL) and [URLSearchParams#getAll](https://developer.mozilla.org/docs/Web/API/URLSearchParams/getAll)
```

Comment:
The method name is included to make it easier to understand the difference between "Get parameter from URL". If you do not need it, please delete it.